hadoop-yarn-project/hadoop-yarn/hadoop-yarn-site/src/site/markdown/TimelineServer.md

```diff
@@ -59,23 +59,23 @@ Generic data is published by the YARN Resource Manager to the timeline store
 Current status
 
 1. The core functionality of the timeline server has been completed.
-1. It works in both secure and non secure clusters.
-1. The generic history service is built on the timeline store.
-1. The history can be stored in memory or in a leveldb database store;
+2. It works in both secure and non secure clusters.
+3. The generic history service is built on the timeline store.
+4. The history can be stored in memory or in a leveldb database store;
   the latter ensures the history is preserved over Timeline Server restarts.
-1. The ability to install framework specific UIs in YARN is not supported.
-1. Application specific information is only available via RESTful APIs using JSON type content.
-1. The "Timeline Server v1" REST API has been declared one of the REST APIs
+5. The ability to install framework specific UIs in YARN is not supported.
+6. Application specific information is only available via RESTful APIs using JSON type content.
+7. The "Timeline Server v1" REST API has been declared one of the REST APIs
   whose compatibility will be maintained in future releases.
-1. The single-server implementation of the Timeline Server places a limit on
+8. The single-server implementation of the Timeline Server places a limit on
   the scalability of the service; it also prevents the service being
   High-Availability component of the YARN infrastructure.
 
 Future Plans
 
 1. Future releases will introduce a next generation timeline service
 which is scalable and reliable, ["Timeline Service v2"](./TimelineServiceV2.html).
-1. The expanded features of this service *may not* be available to
+2. The expanded features of this service *may not* be available to
```

Comment:
this sounds correct only, the numbering is correct only
http://apache.github.io/hadoop/hadoop-yarn/hadoop-yarn-site/TimelineServer.html